Traceback (most recent call last):
  File "/home/thomas/Notebook/anaconda/envs/echem/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/thomas/Notebook/anaconda/envs/echem/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/thomas/Notebook/anaconda/envs/echem/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/thomas/Notebook/anaconda/envs/echem/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/home/thomas/Notebook/anaconda/envs/echem/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/thomas/Notebook/anaconda/envs/echem/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/home/thomas/Notebook/anaconda/envs/echem/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/thomas/Notebook/anaconda/envs/echem/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
xtb_grad_drv = vlx.XtbGradientDriver(xtbdrv)
xtb_opt_drv = vlx.OptimizationDriver(xtb_grad_drv)
xtb_opt_cafestol = xtb_opt_drv.compute(molecule)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mAttributeError[0m                            Traceback (most recent call last)
Cell [0;32mIn[15], line 1[0m
[0;32m----> 1[0m xtb_grad_drv [38;5;241m=[39m [43mvlx[49m[38;5;241;43m.[39;49m[43mXtbGradientDriver[49m[43m([49m[43mxtbdrv[49m[43m)[49m
[1;32m      2[0m xtb_opt_drv [38;5;241m=[39m vlx[38;5;241m.[39mOptimizationDriver(xtb_grad_drv)
[1;32m      3[0m xtb_opt_cafestol [38;5;241m=[39m xtb_opt_drv[38;5;241m.[39mcompute(molecule)

File [0;32m~/Notebook/anaconda/envs/echem/lib/python3.10/site-packages/veloxchem/xtbgradientdriver.py:48[0m, in [0;36mXtbGradientDriver.__init__[0;34m(self, comm, ostream)[0m
[1;32m     43[0m [38;5;28;01mdef[39;00m [38;5;21m__init__[39m([38;5;28mself[39m, comm[38;5;241m=[39m[38;5;28;01mNone[39;00m, ostream[38;5;241m=[39m[38;5;28;01mNone[39;00m):
[1;32m     44[0m [38;5;250m    [39m[38;5;124;03m"""[39;00m
[1;32m     45[0m [38;5;124;03m    Initializes XTB gradient driver.[39;00m
[1;32m     46[0m [38;5;124;03m    """[39;00m
[0;32m---> 48[0m     [38;5;28;43msuper[39;49m[43m([49m[43m)[49m[38;5;241;43m.[39;49m[38;5;21;43m__init__[39;49m[43m([49m[43mcomm[49m[43m,[49m[43m [49m[43mostream[49m[43m)[49m
[1;32m     50[0m     [38;5;28mself[39m[38;5;241m.[39mflag [38;5;241m=[39m [38;5;124m'[39m[38;5;124mXTB Gradient Driver[39m[38;5;124m'[39m

File [0;32m~/Notebook/anaconda/envs/echem/lib/python3.10/site-packages/veloxchem/gradientdriver.py:68[0m, in [0;36mGradientDriver.__init__[0;34m(self, comm, ostream)[0m
[1;32m     65[0m     comm [38;5;241m=[39m MPI[38;5;241m.[39mCOMM_WORLD
[1;32m     67[0m [38;5;28;01mif[39;00m ostream [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m:
[0;32m---> 68[0m     [38;5;28;01mif[39;00m [43mcomm[49m[38;5;241;43m.[39;49m[43mGet_rank[49m() [38;5;241m==[39m mpi_master():
[1;32m     69[0m         ostream [38;5;241m=[39m OutputStream(sys[38;5;241m.[39mstdout)
[1;32m     70[0m     [38;5;28;01melse[39;00m:

[0;31mAttributeError[0m: 'veloxchem.veloxchemlib.XtbDriver' object has no attribute 'Get_rank'
AttributeError: 'veloxchem.veloxchemlib.XtbDriver' object has no attribute 'Get_rank'

